### PR TITLE
Redesign the thanks footer

### PR DESCRIPTION
**Pull request by danrabbit**
# BEFORE
Screenshot from 2019-10-07 13 19 25@2x

# AFTER
Screenshot from 2019-10-07 13 19 04@2x

### CODE REVIEW

File: _includes/thanks.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
+{% unless page.thanks %}
+<div class="thanks">
+  <hr/>
+
+  <p>Thanks to our supporters, backers, and customers! Your contribution makes elementary possible</p>
```
Comment (cassidyjames) [inline _includes/thanks.html]:
```suggestion
  <p>Thanks to our supporters, backers, and customers! Your contribution makes elementary possible.</p>
```

File: _includes/thanks.html [hunk 2]
```diff
@@ -0,0 +1,58 @@
+{% unless page.thanks %}
+<div class="thanks">
+  <hr/>
+
+  <p>Thanks to our supporters, backers, and customers! Your contributions make elementary possible.</p>
+
+  <div class="links"</div>
```
Comment (cassidyjames) [inline _includes/thanks.html]: Oops 

```suggestion
  <div class="links">
```

File: _sass/_layout.scss [hunk 3]
```diff
@@ -81,6 +81,22 @@ article {
...
+
+  .thanks {
+    font-size: 0.8em;
+    font-style: italic;
+    text-align: center;
+
+    .links {
+        column-gap: 3em;
+        display: flex;
+        justify-content: center;
```
Comment (cassidyjames) [inline _sass/_layout.scss]: Something doesn't seem to be working right with this in Epiphany stable or nightly, and so I'd assume Safari as well.

Comment (cassidyjames) [reply _sass/_layout.scss]: Screenshot from 2019-10-07 14-53-04

Comment (danrabbit) [reply _sass/_layout.scss]: Oh, it's probably the `column-gap` property. I'll have to change that to just margins

**Pull request merged**